diff --git a/node_modules/@radix-ui/react-tooltip/dist/index.js b/node_modules/@radix-ui/react-tooltip/dist/index.js
index 2d0d314..95a5af4 100644
--- a/node_modules/@radix-ui/react-tooltip/dist/index.js
+++ b/node_modules/@radix-ui/react-tooltip/dist/index.js
@@ -246,8 +246,8 @@ var TooltipTrigger = React.forwardRef(
           isPointerDownRef.current = true;
           document.addEventListener("pointerup", handlePointerUp, { once: true });
         }),
-        onFocus: (0, import_primitive.composeEventHandlers)(props.onFocus, () => {
-          if (!isPointerDownRef.current) context.onOpen();
+        onFocus: (0, import_primitive.composeEventHandlers)(props.onFocus, (event) => {
+          if (!isPointerDownRef.current && event.currentTarget.matches?.(":focus-visible")) context.onOpen();
         }),
         onBlur: (0, import_primitive.composeEventHandlers)(props.onBlur, context.onClose),
         onClick: (0, import_primitive.composeEventHandlers)(props.onClick, context.onClose)
diff --git a/node_modules/@radix-ui/react-tooltip/dist/index.mjs b/node_modules/@radix-ui/react-tooltip/dist/index.mjs
index 568389b..54a3860 100644
--- a/node_modules/@radix-ui/react-tooltip/dist/index.mjs
+++ b/node_modules/@radix-ui/react-tooltip/dist/index.mjs
@@ -199,8 +199,8 @@ var TooltipTrigger = React.forwardRef(
           isPointerDownRef.current = true;
           document.addEventListener("pointerup", handlePointerUp, { once: true });
         }),
-        onFocus: composeEventHandlers(props.onFocus, () => {
-          if (!isPointerDownRef.current) context.onOpen();
+        onFocus: composeEventHandlers(props.onFocus, (event) => {
+          if (!isPointerDownRef.current && event.currentTarget.matches?.(":focus-visible")) context.onOpen();
         }),
         onBlur: composeEventHandlers(props.onBlur, context.onClose),
         onClick: composeEventHandlers(props.onClick, context.onClose)
